cves/kernel/CVE-2016-6156.yml

```diff
@@ -133,10 +136,10 @@ unit_tested:
 
     For the fix_answer below, check if the fix for the vulnerability involves
     adding or improving an automated test to ensure this doesn't happen again.
-  code:
-  code_answer:
-  fix:
-  fix_answer:
+  code: false
+  code_answer: no automated unit tests were found
```

Comment:
It's nitpicky, but capitalizing the first letter and adding a period would maintain consistent formatting.